roles/aks/tasks/tavros_config_backup.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Store Tavros Vars
```

Comment:
we also need to store the ssh keys

Reply:
I guess we should do the same for aws at the same time. Perhaps a second PR to keep it together after this, and backups are merged?

Reply:
ah I thought we were already backing up the kops ssh keys, so yeah you can ignore it here and we can add both in a nother PR